contracts/SnapshotToken/Extensions/SimpleVote.sol

```diff
@@ -0,0 +1,191 @@
+pragma solidity 0.4.26;
+
+import "./ISnapshotableToken.sol";
+/* import "../../Math.sol"; */
+
+/// Contract to allow weighted voting based on a snapshotable token (with relayed, batched voting)
+contract SimpleVote {
+
+    ////////////////////////
+    // Types
+    ////////////////////////
+    struct Proposal {
+        uint256 inFavor;
+        uint256 against;
+        uint256 endTime;
+        address owner;
+        uint256 snapshotId;
+        mapping (address => bool) hasVoted;
+    }
+
+    /////////////////////////
+    // Immutable state
+    ////////////////////////
+    ISnapshotableToken public TOKEN;
+    // % of tokens to participate in a voting to make it valid
+    uint256 public QUORUM = 40;
+    uint256 public VOTING_PERIOD;
+
+    /////////////////////////
+    // Mutable state
+    ////////////////////////
+    mapping (uint256 => Proposal) private _proposals;
```

Comment:
I thought one way of learning about hte proposal (events) is enough. The second just wastes gas. so yeah, this was by design.... if we acually planned to build a vote-ui this could be reconsidered, but so far the design philosophy is to save gas